### DISCUSSION TITLE
module 'fipy.tools.numerix' has no attribute 'obj2sctype'

### DISCUSSION THREAD

**Discussion by Fiki-chemE** [usnistgov/fipy#1053] [Q&A]
I have tried upgrading and downgrading both fipy and numerix with no success. Has anyone been able to bypass this problem?

**Comment by guyer** [comment]
You need to either install numpy < 2.0, install our [issue1049-Numpy_2.0.0_breaks_things](https://github.com/usnistgov/fipy/tree/issue1049-Numpy_2.0.0_breaks_things) branch, or wait for us to merge #1050

**Reply by guyer** [reply]
[FiPy 3.4.5](https://github.com/usnistgov/fipy/releases/tag/3.4.5) has been released, which fixes this issue
